schema/360-giving-schema.json

```diff
@@ -926,31 +926,31 @@
         },
         "streetAddress": {
           "type": "string",
-          "description": "Building number and street name.",
+          "description": "Recipient organisation address building number and street name.",
```

Comment:
I think these fields are shared by Recipient and Funder organisation? If so, these need to be edited to removed 'Recipient from the start of all descriptions - so it will be generic.